mxcubeweb/core/components/user/usermanager.py

```diff
@@ -150,7 +193,15 @@ def update_operator(self, new_login=False):
                     elif _u.selected_proposal is not None:
                         self.app.lims.select_session(_u.selected_proposal)
 
-    def is_inhouse_user(self, user_id):
+    def is_inhouse_user(self, user_id: str) -> bool:
+        """Check if the ``user_id`` is in the in-house user list.
+
+        Attributes:
+            user_id: user id composed from code and number.
```

Comment:
I guess the attribute description has to be on a different line.

At least it does not render any attribute description currently: <https://mxcubeweb--1561.org.readthedocs.build/en/1561/dev/autosummary/mxcubeweb.core.components.user.usermanager.html#mxcubeweb.core.components.user.usermanager.BaseUserManager.is_inhouse_user>.

```suggestion
        Attributes:
            user_id:
                User ID composed from code and number.
```